src/main/java/com/hackclub/hccore/advancements/MasterMobSlayerAdv.java

```diff
@@ -0,0 +1,74 @@
+package com.hackclub.hccore.advancements;
+
+import com.fren_gor.ultimateAdvancementAPI.advancement.Advancement;
+import com.fren_gor.ultimateAdvancementAPI.advancement.BaseAdvancement;
+import com.fren_gor.ultimateAdvancementAPI.advancement.display.AdvancementDisplay;
+import com.fren_gor.ultimateAdvancementAPI.advancement.display.AdvancementDisplayBuilder;
+import com.fren_gor.ultimateAdvancementAPI.util.AdvancementKey;
+import com.fren_gor.ultimateAdvancementAPI.util.CoordAdapter;
+import org.bukkit.Material;
+import org.bukkit.entity.EntityType;
+import org.bukkit.event.entity.EntityDeathEvent;
+
+public class MasterMobSlayerAdv extends BaseAdvancement {
+
+  static final AdvancementDisplayBuilder<AdvancementDisplay.Builder, AdvancementDisplay> displayBuilder = new AdvancementDisplay.Builder(
+      Material.NETHERITE_AXE, "Master Mob Slayer")
+      .challengeFrame()
+      .announceChat()
+      .showToast()
+      .description("Kill EVERY hostile Mob 10 times");
+  static final int maxProgression = 640; // 10 times for each of the 64 hostile mobs
```

Comment:
Wouldn't this mean that I could just kill 640 endermen in a farm and get the advancement?

Reply:
i dont know
i thought it meant that you need to kill 10 of each hostile mob to get the advancement

Reply:
Hmm I haven't tested it, but from what I see, it's not tracking how many of each mob you've killed, just how many mobs in total

Reply:
@polypixeldev i dont know how to make it count 10 of each mob death tho...

Reply:
Yeah, this is a bit of a tough one to do. The only way I can think of would be to add a map to the PlayerData class which stores how many of each mob they've killed, and update it accordingly in the EntityDeathEvent handler.

Reply:
you don't really need a map as player stats are already stored